src/Lykke.Logs/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -12,6 +14,7 @@ public class LykkeLogToAzureSlackNotificationsManager : ProducerConsumer<LogEnti
         private readonly ISlackNotificationsSender _slackNotificationsSender;
         private readonly ILog _lastResortLog;
         private readonly string _component;
+        private readonly IEnumerable<string> _logLevels;
```

Comment:
I would use HashSet for better performance.

Reply:
Not always better for small integer arrays. GetHashCode() can cost more then iterating through the array